.githooks/pre-commit

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+early_exit=0
+
+# Only run if there are files staged for commit
+changed_files=$(git diff --cached --name-only --diff-filter=ACM | paste -s -d' ' -)
```

Comment:
Is paste a standard command? Not sure if macs have it

Reply:
No clue :)  is it on yours? `which paste`

Reply:
Oh right you're also not on a Mac :woman_facepalming: